eng/build.ps1

```diff
@@ -57,6 +58,11 @@ if ($MyInvocation.InvocationName -eq ".") {
   exit 0
 }
```

Comment:
Are we still dot sourcing this script anywhere? If not we can remove the if block.

Reply:
Hmmm I'm not sure about that. I don't think so. Would searching for `-Command` be enough?

Reply:
I'm relatively sure that this is dead code and you can just remove it.

Reply:
Ok. Will remove it then.

Reply:
thanks